liquibase-integration-tests/src/test/java/liquibase/dbtest/AbstractIntegrationTest.java

```diff
@@ -565,17 +566,22 @@ public void testUpdateClearUpdate() throws Exception {
 
     @Test
     @SuppressWarnings("squid:S2699") // Successful execution qualifies as test success.
-    public void testRollbackableChangeLog() throws Exception {
+    public synchronized void testRollbackableChangeLog() throws Exception {
```

Comment:
Just to be sure add synchronized, for static property executeCallCount and rollbackCallCount.

Reply:
This is a test, I'm not sure to understand why it would need a sync keyword. I don't think this is needed here.

Reply:
how to care for static property?
maybe, alternative way was check call count the `execute` and `rollback` method with Mockito. I think that's the way to do it politely.

Reply:
I found alternative way. it is Check ExampleCustomTaskChange with executed `liquibase.getDatabaseChangeLog().getChangeSets()`.
I will try it.

Reply:
```java
        List<CustomChange> customChanges = liquibase.getDatabaseChangeLog().getChangeSets().stream().flatMap(changeSet ->
                changeSet.getChanges().stream().filter(change ->
                        change instanceof CustomChangeWrapper &&
                                ((CustomChangeWrapper) change).getCustomChange() instanceof ExampleCustomTaskChange
                ).map(change ->
                        ((CustomChangeWrapper) change).getCustomChange())
        ).collect(Collectors.toList());
        for (CustomChange customChange : customChanges) {
            assertEquals(1, ((ExampleCustomTaskChange)customChange).executeCallCount);
            assertEquals(0, ((ExampleCustomTaskChange)customChange).rollbackCallCount);
        }
```

I wrote this code. but cannot check call-count, actual value is `0`. because, difference ExampleCustomTaskChange object.

Reply:
I was going to suggest making ExampleCustomTaskChange a singleton class, and then checking `rollbackcount` but I think this won't help with this issue. Let me ask tomorrow one of our team members with more experience/knowledge on custom changes because my guess is rollback is executing twice because there might be something else calling also this same method.

Reply:
Hi @momosetkn,

I was talking about this with someone else from the team, and we agreed that `synchronized` keyword is not needed there. To validate counts, we think maybe what we can do here is to have a map and save your `ExampleCustomTaskChange` instances there, then save this map in the current scope.  I was thinking of adding something like this in `CustomChangeWrapper`, but I'm unsure if this is the right place. It would probably be if some other custom changes implement something like this, but for now, we can probably do this in the scope you were trying to do this validation.

Thanks,
Daniel.

Reply:
I see, ThreadLocal(map in the current scope) can resolve synchronous problem.
I fix.

Reply:
Sorry for the late fix.
I fixed.

Reply:
@MalloD12 
Sorry for the late fix.
I fixed.